operational/flexera/msp/parent_child_group_sync/flexera_msp_parent_child_group_sync.pt

```diff
@@ -0,0 +1,1262 @@
+  type "string"
+  category "Policy Settings"
+  label "Prefix String"
+  description "Prefix to indicate the Group should be synced.  This is the first part of the string that needs to be in the description. **`<PrefixString>`**_`<ChildOrgID>`_`Child Group Name`"
+  default "GroupSync"
+end
+
+parameter "param_invitation_flow" do
+  type "string"
+  category "Policy Settings"
+  label "Invitation Flow"
+  description "Disable/Enable the invitation flow.  Users that can't be added directly to the Org due to missing required Identity Provider configuration will raise an error."
+  allowed_values "User Invitations Disabled", "User Invitations Enabled"
+  default "User Invitations Disabled"
+end
+
+parameter "param_batch_count" do
+  type "number"
+  category "Policy Settings"
+  label "Batch Count"
+  description "If Batch Count is > 1 then Orgs will be synced incrementally over multiple runs in batches.  This is intended to be used if you are hitting timeout issues during applied policy evaluation."
+  min_value 1
+  default 1
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+credentials "auth_flexera" do
+  schemes "oauth2"
+  label "Flexera"
+  description "Select Flexera One OAuth2 credentials"
+  tags "provider=flexera"
+end
+
+###############################################################################
+# Pagination
+###############################################################################
+
+pagination "pagination_flexera" do
+  get_page_marker do
+    body_path jmes_path(response, "nextPage")
+  end
+  set_page_marker do
+    uri true
+  end
+end
+
+###############################################################################
+# Datasources & Scripts
+###############################################################################
+
+# Get applied policy metadata for use later
+datasource "ds_applied_policy" do
+  request do
+    auth $auth_flexera
+    host val($ds_flexera_api_hosts, 'flexera')
+    path join(["/policy/v1/orgs/", rs_org_id, "/projects/", rs_project_id, "/applied-policies/", policy_id])
+    ignore_status 404 # Ignore 404 returned when we are running this with retrieve_data
+  end
+end
+
+# Get region-specific Flexera API endpoints
+datasource "ds_flexera_api_hosts" do
+  run_script $js_flexera_api_hosts, rs_optima_host
+end
+
+script "js_flexera_api_hosts", type: "javascript" do
+  parameters "rs_optima_host"
+  result "result"
+  code <<-EOS
+  host_table = {
+    "api.optima.flexeraeng.com": {
+      flexera: "api.flexera.com",
+      fsm: "api.fsm.flexeraeng.com"
+    },
+    "api.optima-eu.flexeraeng.com": {
+      flexera: "api.flexera.eu",
+      fsm: "api.fsm-eu.flexeraeng.com"
+    },
+    "api.optima-apac.flexeraeng.com": {
+      flexera: "api.flexera.au",
+      fsm: "api.fsm-apac.flexeraeng.com"
+    }
+  }
+
+  result = host_table[rs_optima_host]
+EOS
+end
+
```

Comment:
This is the start delimitor for the "context" incident, which can be used to hold state between run. This code between "Begin" (Line 140) + "End" Context comments (Line 230) is mostly reusable code that çan be copied and pasted into other PTs, and there's comments where to edit if needed.  This is a new capability, and we need to decide if it's something we want to re-use in other Policy Templates.  For now, it's needed in this one to maintain the batch index -- which batch number was last run.